assignment5/AlphabetFinder.java

```diff
@@ -0,0 +1,144 @@
+package assignment5;

```

Comment:
Really nice code, well done!

If I was to suggest an improvement it would be in test cases and handling for illegal input.
It would be nice to have test cases for if the user entered "ab", "ac", "bc", "bb" as input. Its important that the code doesn't infinite loop in this sort of case, but instead gives a readable exception.